From 06b24089986523806d386b9e3cfa4fcf5eeb87e6 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 17 Mar 2015 12:53:29 +0000
Subject: [PATCH 2/2] Don't bother checking return from virInitialize.

The Perl bindings don't do this, and it seems that the call can never
fail, or if it does we don't care.
---
 libvirt/libvirt_c_oneoffs.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/libvirt/libvirt_c_oneoffs.c b/libvirt/libvirt_c_oneoffs.c
index 32e5a4b..5d82194 100644
--- a/libvirt/libvirt_c_oneoffs.c
+++ b/libvirt/libvirt_c_oneoffs.c
@@ -1229,12 +1229,9 @@ CAMLprim value
 ocaml_libvirt_init (value unit)
 {
   CAMLparam1 (unit);
-  CAMLlocal1 (rv);
-  int r;
 
   virSetErrorFunc (NULL, ignore_errors);
-  r = virInitialize ();
-  CHECK_ERROR (r == -1, "virInitialize");
+  virInitialize ();
 
   CAMLreturn (Val_unit);
 }
-- 
2.3.1

